app/controllers/admin/communities_controller.rb

```diff
@@ -221,4 +222,8 @@ def payment_settings_api
     TransactionService::API::Api.settings
   end
 
+  def escrow_payments?(community)
+    MarketplaceService::Community::Query.payment_type(community) == :braintree
```

Comment:
Is it intentional that you pass the whole community model instead of just the id?

Community.find_by_id(@current_community) handles this well of course.